src/components/Accordion/AccordionHeader.module.css

```diff
@@ -10,6 +10,27 @@
   background-color: var(--component-accordion_header-color-background);
 }
 
+/* breakpoints-xs */
+@media only screen and (min-width: 0) {
+  .accordion-header__subtitle {
+    display: none;
+  }
+}
+
+/* breakpoints-sm */
+@media only screen and (min-width: 540px) {
+  .accordion-header__subtitle {
+    opacity: 60%;
+    font-size: 1.4rem;
+    display: block;
+  }
+  .accordion-header--subtitle {
+    padding-top: 0.3rem;
+    padding-bottom: 0.3rem;
+    padding-right: 0.3rem;
+  }
```

Comment:
This will make the accordions that has a subtitle bigger than the ones that doesnt. Are we sure this is right? It will look odd in a list where some items have subtitle, and others dont. This padding should perhaps just always be added instead?

Reply:
I just figured it looked too cramped if I didn't add extra padding and the subtitle was enabled. I figured it would look strange to have subtitles on only some accordions in a section anyway.
If this padding should be there all the time, I don't know. It could be, I suppose, but I didn't dare remove the tokens that were already there.
Maybe UX would have an opinion on this?

Reply:
I agree it does look a bit cramped :) And you may be right that it is unlikely to have some accordions in a group that has a subtitle, and others do. But theres nothing preventing this, and if it happens it will look odd because of the different sizes. Would be good to reach out to the responsible UX person to clarify.

Reply:
Talked to Hanne and she said to expand when using subtitles. She said that's what is done in production already. 🙂